### DISCUSSION TITLE
how can we train the "train" dataset

### DISCUSSION THREAD

**Discussion by james201011071** [pierotofy/OpenSplat#26] [General]
hi, i test the banana dataset, but how can we test the "train" dataset, 30K iterations seems do not work well. can you give me some suggestions?

**Comment by yuancaimaiyi** [comment]
@pierotofy  the same question ,the train data gs's result as follow:
![2024-02-27 17-48-19屏幕截图](https://github.com/pierotofy/OpenSplat/assets/28293910/34323355-5457-463f-8c64-0aa0062fd5b3) ,the num-iters set to 30000

**Comment by pierotofy** [comment]
The train dataset is quite noisy; if you zoom in to the train location, you should be able to see it. I used https://playcanvas.com/supersplat/editor in the past to crop out the noise before displaying it.

**Reply by ChenShujuanCN** [reply]
I am so sorry to bother you. I want to editor the sparse_pc.ply as you said, but got a wrong.Can you give me some suggestions?
![屏幕截图 2025-02-06 125939](https://github.com/user-attachments/assets/eb875434-6eda-47e4-bc35-6567f77a9842)

**Comment by pierotofy** [comment]
https://github.com/pierotofy/OpenSplat/issues/24 could be implemented to make it easier to locate the subjects of a scene.

**Comment by pierotofy** [comment]
I rendered this with 7k iterations:

![image](https://github.com/pierotofy/OpenSplat/assets/1951843/089109c5-151f-4454-8f01-f1b3065581d6)

**Reply by james201011071** [reply]
yes, thank you for your patient reply. i want to know what kind of reason why the "trains" dataset has the noise. maybe the initial sfm points problem caused by the colmap?
